packages/zosfiles/__tests__/__system__/methods/download/__resources__/upload_file_to_uss.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+account=$1
+host=$2
+port=$3
+user=$4
+password=$5
+ru=$6
+inputFile=$7
+ussName=$8
+encoding=$9
+
+zowe zos-files upload file-to-uss $inputFile $ussName --encoding $encoding --host $host --port $port --user $user --password $password --ru $ru
```

Comment:
Please put quotes around the `$inputfile` and `$ussname`, otherwise system tests will fail if they run in an environment with spaces in the path.